docs/content/en/docs/tasks/packages/_index.md

```diff
@@ -23,7 +23,7 @@ Skip the following installation steps if the returned result is not empty.
     ```bash
     eksctl anywhere version
     ```
-* Make sure cert-manager is up and running in the cluster.
+* Make sure cert-manager is up and running in the cluster. Note cert-manager is not installed on workload clusters by default. If cert-manager is not installed, you can manually install cert-manager with Helm 3.8+ and follow the instructions below to finish the package controller installation.
```

Comment:
There are no instructions below

Reply:
I'm referring to the `Install package controller` instructions below. As I mentioned in my other comment, I removed the cert-manager installations for now given we don't have a version that is ready to be used. I will add such instructions back once available!

Reply:
```suggestion
* Make sure cert-manager is up and running in the cluster. Note cert-manager is not installed on workload clusters by default. If cert-manager is not installed, you can manually install cert-manager and follow the instructions below to finish the package controller installation.
```